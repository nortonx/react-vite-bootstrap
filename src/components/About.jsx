import React from "react";

const About = () => {
  return(
    <div className="content">
      <h1>This is the About Page</h1>
    </div>
  )
}

export default About;