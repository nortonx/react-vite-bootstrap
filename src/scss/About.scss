.about {
  height: 72vh;
}