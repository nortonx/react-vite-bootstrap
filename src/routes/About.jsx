import "../scss/About.scss";

const About = () => {
  return(
    <div className="container about" data-testid="about">
      <h1>This is the About content</h1>
    </div>
  );
};

export default About;