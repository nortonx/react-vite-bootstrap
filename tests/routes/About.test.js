import { render, screen } from "@testing-library/react";
import renderer from "react-test-renderer";

import About from "../../src/routes/About";

describe("About route", () => {
  it("Should render the component and update snapshot", async () => {
    render(<About />);
    const element = await screen.findByTestId("about");
    expect(element).toBeInTheDocument();
    const about = renderer.create(<About />);
    expect(about).toMatchSnapshot();
  });
})