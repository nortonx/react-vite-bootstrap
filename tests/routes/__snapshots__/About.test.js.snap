// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About route Should render the component and update snapshot 1`] = `
<div
  className="container about"
  data-testid="about"
>
  <h1>
    This is the About content
  </h1>
</div>
`;
